dockerfiles/init/manifests/che.env

```diff
@@ -101,7 +101,16 @@
 #     agents within the workspace when it boots, then the workspace will not be started.
 #CHE_MACHINE_WS_AGENT_MAX_START_TIME_MS=300000
 
-
+# Workspace Agent Java Options
+#   Java command line options will be exported as a value of the JAVA_OPTS
```

Comment:
Please use this comment.  I think we should avoid the idea of "export" because to an end user, an "export" is something that comes *out* of the workspace.  When I think we are really talking about taking parameters from one location and injecting them into another location.

```
Java command line options to be added to JVM's that run agents within workspaces.
```